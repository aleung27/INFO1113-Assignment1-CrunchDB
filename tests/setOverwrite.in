SET a 3
LIST ENTRIES
SET a 99
LIST ENTRIES
SET a 0 9 3 1 4 1 1 3 5 
LIST ENTRIES
SET a 0
LIST ENTRIES
