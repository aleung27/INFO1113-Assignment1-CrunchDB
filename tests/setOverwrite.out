> ok

> a [3]

> ok

> a [99]

> ok

> a [0 9 3 1 4 1 1 3 5]

> ok

> a [0]

